### ISSUE TITLE
Gender accuracy in lfw is very low

### ISSUE THREAD

**Issue by xudou3** [oarriaga/face_classification#109]
Hi, i test this model with a little bit modify image_emotion_gender_demo.py to classify gender laber on data set lfw(http://vis-www.cs.umass.edu/lfw/) and the accuracy for female is 82% and for male is just 69%. 
What I have done to the code is just make image_path in a for loop to read all file in a floder. And there was an error occured when load rgb image in utils/inference.py in function load_image with unmatched parameters so I fix it. 
I am using python2.7 and tensorflow1.8.0 with GPU support.

**Comment by oarriaga** [comment]
Hi thank you for your comment. I am actually impressed on how ``high`` it is. I have discussed previously (and within the paper) that the "sex" classifier is biased given the data in which was trained on.
